Signal/src/environment/PropertyListPreferences.m

```diff
@@ -130,9 +120,21 @@ - (void)setHasRegisteredVOIPPush:(BOOL)enabled
     [self setValueForKey:PropertyListPreferencesKeyHasRegisteredVoipPush toValue:@(enabled)];
 }
 
+- (BOOL)loggingIsEnabled
+{
+    NSNumber *preference = [NSUserDefaults.standardUserDefaults objectForKey:PropertyListPreferencesKeyEnableDebugLog];
```

Comment:
That seems good.  

Hmm, so the migration happens _after_ the "is logging enabled" preference is checked at app startup? I see you turn off logging at that time. Since no PII or user activity is logged by that point - maybe just a line or two about the current app version and the "app launched" event, that seems fine.